File: Beginner/Tickets (TICKETS5)/tickets.cpp
#include <iostream>
#include <string>
using namespace std;


int main() {

    int t;
    string s;
    cin >> t;

    while (t--) {
        cin>>s;

        int i, f=0, l=s.length();
	    if(s[0]==s[1]){
	        f = 1;
	        }
	    for(i=0;i<l-2;i++) {
	        if(s[i]!=s[i+2]){
	            f = 1;
	            break;
	        }
	    }
	    if(f)
	     cout<<"NO\n";
	    else
	     cout<<"YES\n";
    }
    

    return 0;
}

File: Beginner/Tickets (TICKETS5)/tickets.py
t = int(input())

while t:
    s = input()
    if len(set(s))==2:
        for j in range(0,len(s)-1):
            if s[j]==s[j+1]:
                print("NO")
                break
        else:
            print("YES")
    else:
        print("NO")